fetchai/ledger/api/tx.py

```diff
@@ -76,14 +76,14 @@ def __init__(self,
         self.valid_until = valid_until
         self.charge = charge
         self.charge_limit = charge_limit
-        self.transfers = {Address(t['to']): t['amount'] for t in transfers}
+        self.transfers = [{Address(t['to']): int(t['amount'])} for t in transfers]
```

Comment:
This should be a list of Transfer objects, rather than of dicts

Reply:
I wanted to keep the same api interface (returning list of dictionaries) in order to avoid possible impact on already exiting callers.

Reply:
There is already a change from a single dict to a list of dicts, so this would be breaking regardless

Reply:
I would still change this to a Transfers object - you've already changed the signature, and this usage of a list of dicts is convoluted.

Reply:
Done.